plugins/woocommerce/tests/e2e-pw/tests/merchant/create-order.spec.js

```diff
@@ -181,10 +179,22 @@ test.describe( 'WooCommerce Orders > Add new order', () => {
 			],
 		} );
 		// clean up tax classes and rates
-		for ( let i = 0; i < taxClassSlugs.length; i++ ) {
-			await api.delete( `taxes/classes/${ taxClassSlugs[ i ] }`, {
-				force: true,
-			} );
+		for ( const { slug } of taxClasses ) {
+			await api
+				.delete( `taxes/classes/${ slug }`, {
+					force: true,
+				} )
+				.catch( ( error ) => {
+					if (
+						error.response.data.code ===
+						'woocommerce_rest_invalid_tax_class'
+					) {
+						// do nothing, probably the tax class was not created due to a failing test
+					} else {
+						// Something else went wrong.
+						throw new Error( error.response.data );
+					}
+				} );
```

Comment:
Simplified the loop, and handle the error when something goes wrong with deleting tax classes as they sometimes do.